src/main/resources/static/js/localize-timestamps.js

```diff
@@ -0,0 +1,6 @@
+jQuery(document).ready(() => {
+    jQuery("td.timestamp").each(function() {
+        const dt = new Date(Date.parse($(this).text()));
```

Comment:
Is there a different between `new Date(Date.parse(txt))` and `new Date(txt)`? On [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/parse#Syntax) they make the distinction between explicit and implicit invocations, I'm wondering if the implicit call would cost anything, since it would be shorter and easier to read.

Reply:
It seems that there is not. I simplified it, thanks 👍